tests/virtual_machines_test.go

```diff
@@ -195,7 +195,7 @@ var _ = Describe("Virtual Machines", func() {
 			})
 		})
 		//2162 test postponed due to issue: https://github.com/k8snetworkplumbingwg/kubemacpool/issues/104
-		PContext("When trying to create a VM after all MAC addresses in range have been occupied", func() {
+		Context("When trying to create a VM after all MAC addresses in range have been occupied", func() {
```

Comment:
please keep  the P on these tests as it is currently examined by @alonSadan in another [task](https://issues.redhat.com/browse/CNV-4404)